frontend/src/app/shared/components/header/header.component.html

```diff
@@ -0,0 +1,73 @@
+<link href="https://fonts.googleapis.com/css?family=Inter" rel="stylesheet" />
+
+<header>
+    <div class="logo"><img src="assets/header/header-logo.png" /></div>
+    <div class="nav">
+        <div class="tab tab-active">
+            <a routerLink="#">Availability</a>
+        </div>
+        <div class="tab">
+            <a routerLink="#">Bookings</a>
+        </div>
+    </div>
+    <div class="button">
+        <div class="dropdown">
+            <button class="drop-icon hover-drop-icon">
+                <div class="label" (click)="onSelectDropdown(1)">
+                    Binary Team<img class="pop-up-icon" src="assets/header/header-pop-up.png" />
+                </div>
+            </button>
+            <div class="dropdown-content" id="dropdown-content-1">
+                <a class="dropdown-content__item" routerLink="#">
+                    <img src="assets/header/header-pop-up__team.png" /><span class="dropdown-content__label"
+                        >Binary Team</span
+                    >
+                </a>
+                <a class="dropdown-content__item" routerLink="#">
+                    <img src="assets/header/header-pop-up__team.png" /><span class="dropdown-content__label"
+                        >Team 2</span
+                    >
+                </a>
+                <a class="dropdown-content__item" routerLink="#">
+                    <img src="assets/header/header-pop-up__team.png" /><span class="dropdown-content__label"
+                        >Team 3</span
+                    >
+                </a>
+                <a class="dropdown-content__btn" routerLink="#">
+                    <img src="assets/header/header-pop-up__plus.png" /><span class="dropdown-content__label"
+                        >New Team</span
+                    >
+                </a>
+            </div>
+        </div>
+    </div>
+    <div style="left: 88%" class="button">
+        <div class="dropdown">
+            <button class="drop-icon">
+                <div class="avatar">
+                    <img (click)="onSelectDropdown(2)" src="assets/header/header-ava-template.png" />
+                </div>
+            </button>
+            <div class="dropdown-content" id="dropdown-content-2">
+                <a class="dropdown-content__item" routerLink="#">
+                    <img src="assets/header/header-pop-up__account.png" /><span class="dropdown-content__label"
+                        >Account</span
```

Comment:
```suggestion
                    <img src="assets/header/header-pop-up__account.png" />
                    <span class="dropdown-content__label">Account</span
```

Do the same in all places. Your way is more React code styling